.semaphore/semaphore.yml

```diff
@@ -47,7 +55,9 @@ blocks:
           - echo $SEMAPHORE_WORKFLOW_ID
           - sem-version java 17
           - checkout
-          - cache restore
+          - artifact pull workflow cache-post-install.tgz
+          - tar zxvf cache-post-install.tgz
```

Comment:
same here, let's remove the 'v' argument to reduce build verbosity

Reply:
done